books_scraper/spiders/books.py

```diff
@@ -0,0 +1,65 @@
+import scrapy
+from scrapy.http import Response
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/index.html"]
+
+    @staticmethod
+    def parse_detail_info_from_book(response: Response):
+        ratings = {
+            "One": "1",
+            "Two": "2",
+            "Three": "3",
+            "Four": "4",
+            "Five": "5"
+        }
+        available_text = " ".join(
+            response.css("p.instock.availability::text").getall()).strip()
+        available = ""
+
+        for i in available_text:
+            if i.isdigit():
+                available += i
+
+        rating_word = response.css(
+            "p.star-rating::attr(class)"
+        ).get().split()[-1]
+        rating = ratings.get(rating_word, 0)
```

Comment:
The `ratings.get(rating_word, 0)` will return `0` if the rating word is not found in the dictionary. Ensure this is the intended behavior, or consider logging a warning if an unexpected rating is encountered.